### ISSUE TITLE
pip failing to install pyscf-tblis

### ISSUE THREAD

**Issue by yapolyak** [pyscf/pyscf-tblis#1]
Hi, 

I tried to install this extension to work with pyscf, but pip (latest version) complains:
```
ERROR: Could not find a version that satisfies the requirement pyscf-tblis (from versions: none)
ERROR: No matching distribution found for pyscf-tblis
```
I do the following (from MacOS v.11.5.2, Python==3.9, clean environment):
```
pip install pyscf
pip install pyscf-tblis
```

Many thanks!
Iakov

**Comment by sunqm** [comment]
If `pip install pyscf-tblis` does not work, you can try `pip install git+https://github.com/pyscf/pyscf-tblis` to install from github source code

**Comment by fishjojo** [comment]
The library files are not included after `pip install` somehow.

**Comment by ajz34** [comment]
Following is another different problem when installing :joy_cat:

When I try to build pyscf-tblis recently (0.1.3 build by source), I found that it is quite possible mixing C compiler (as icc) and CXX compiler (as /usr/bin/c++)

https://github.com/pyscf/pyscf-tblis/blob/b8bb984894696dac6aae43151e86ffedee74abfb/pyscf/tblis_einsum/CMakeLists.txt#L50-L51

and then generate the following error:
```
      CXX      src/configs/core2/lib_libcore2_la-config_ker.lo
    c++: error: language SSSE3 not recognized
    c++: error: language SSSE3 not recognized
    make[5]: *** [Makefile:2250: src/configs/core2/lib_libcore2_la-config_ker.lo] Error 1
```

TBLIS compiler flags are decided by C compiler, and compiling work by CXX compiler. So using icc with /usr/bin/c++ will probably pass unrecognizable compile flags (like `-xSSSE3`) to c++ compiler.
TBLIS try to use icc, then gcc, then clang (unless that `CC` or `CXX` is defined in environment variables or specified by cmake).

I suggest that either let TBLIS, or let PySCF cmake, decide **both** C and CXX compilers would be utilized.

**Comment by ajz34** [comment]
@fishjojo I found that's true 😅 
Actually I'm not sure how pip install exactly works; however I found MANIFEST.in:
https://github.com/pyscf/pyscf-tblis/blob/b8bb984894696dac6aae43151e86ffedee74abfb/MANIFEST.in#L10
Adding `*.so` and `*.so.*` to the file list above could resolve this problem (partially). This could copy all libraries required (`libtblis_einsum.so`, `libtblis.so.0`, `libtci.so.0`). But redundant libraries (copied because soft-link, such as `libtblis.so.0.0.0`) are also copied, which situation is not expected but not incurring error.
